chia/data_layer/data_store.py

```diff
@@ -655,3 +749,55 @@ async def get_proof_of_inclusion_by_key(
         async with self.db_wrapper.locked_transaction(lock=lock):
             node = await self.get_node_by_key(key=key, tree_id=tree_id, lock=False)
             return await self.get_proof_of_inclusion_by_hash(node_hash=node.hash, tree_id=tree_id, lock=False)
+
+    async def get_left_to_right_ordering(
+        self,
+        node_hash: bytes32,
+        tree_id: bytes32,
+        root_hash: bytes32,
+        *,
+        lock: bool = True,
+        num_nodes: int = 2500,
+    ) -> Tuple[bool, List[Dict[str, Any]]]:
```

Comment:
Please no returning tuples.  That just leaves us hoping we're really careful to have the return and call in the same order and also leads to nested hints that we can't keep track of.  Can the `bool` not be handled by an exception?  If we really need two things we can write a small class using dataclasses.